packages/synapse-react-client/src/umd.index.ts

```diff
@@ -143,7 +142,6 @@ const SynapseComponents = {
   displayToast,
   IconSvg,
   UserProfileLinks,
-  PlotlyWrapper,
```

Comment:
I think we ran a POC with utilizing this in Agora at one point, but I think they're only using SRC for rendering Synapse Wikis now (@sagely1 or @hallieswan , could you confirm?)

Reply:
[A search in Sage-Bionetworks/sage-monorepo](https://github.com/search?q=repo%3ASage-Bionetworks%2Fsage-monorepo%20synapse-react-client&type=code) tells me that no projects there, including Agora, are currently using `synapse-react-client`.